src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
use interface prefer